LogicalTypes.md

```diff
@@ -144,6 +144,18 @@ example, there is no requirement that a large number of days should be
 expressed as a mix of months and days because there is not a constant
 conversion from days to months.
 
+### INT96 timestamps (also called IMPALA_TIMESTAMP)
+
+_(deprecated)_ Timestamps saved as an `int96` are made up of the nanoseconds in the day
+(first 8 byte) and the Julian day (last 4 bytes). No timezone is attached to this value.
```

Comment:
I think we need to specify the signedness - the first 8 bytes are a uint64 and the last 4 bytes are a int32 I believe